playwright-e2e/dss/component/input.ts

```diff
@@ -43,7 +43,7 @@ export default class Input extends WidgetBase {
 
     const useType = type ? type : false;
     nth ? this.nth = nth : this.nth;
-
+    await this.toLocator().scrollIntoViewIfNeeded();
```

Comment:
On rare occasion, input element is on the very bottom of page and if there is a dropdown, it is not visible.